sections/image-banner.liquid

```diff
@@ -30,16 +30,20 @@
 {%- endstyle -%}
 
 {%- liquid
+  assign full_width = '100vw'
+  assign widths = '375, 550, 750, 1100, 1500, 1780, 2000, 3000, 3840'
+
   if section.settings.image_behavior == 'ambient'
     assign half_width = '60vw'
     assign full_width = '120vw'
     assign stacked_sizes = '(min-width: 750px) 60vw, 120vw'
     assign widths = '450, 660, 900, 1320, 1800, 2136, 2400, 3600, 7680'
+  elsif section.settings.image_behavior == 'fixed'
+    assign half_width = '100vw'
+    assign stacked_sizes = '(min-width: 750px) 100vw'
```

Comment:
It seems to load properly an image that is `1100` wide when I check but I think the `stacked_sizes` here should not need the `min-width: 750px` check, no ?

Reply:
Updated to `100vw`.